presto-main/src/main/java/com/facebook/presto/sql/ExpressionUtils.java

```diff
@@ -243,6 +250,14 @@ public static Expression stripDeterministicConjuncts(Expression expression)
                 .collect(toImmutableList()));
     }
 
+    public static Expression stripDynamicFilters(Expression expression)
```

Comment:
I like it. Will do as a separate commit, so we can make the commit `Implement dynamic filtering pushdown` smaller.